config/samples/infrastructure_v1beta1_ibmpowervscluster.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: IBMPowerVSCluster
+metadata:
+  name: ibmpowervscluster-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
wondering if we can add something here, this comment applies for the remaining samples as well.

Reply:
Done